sdk/monitor/opentelemetry-exporter/src/Declarations/Contracts/Generated/ContextTagKeys.ts

```diff
@@ -0,0 +1,153 @@
+// THIS FILE WAS AUTOGENERATED
+
+class ContextTagKeys {
```

Comment:
I get that this file is autogenerated - but why do you have a class that contains only public fields? what's the point of a class without a single method? Couldn't this just be a TS interface?

Reply:
These autogenerated classes need to be updated, but I think the thought was to have a similar usage pattern across all SDK languages; `new ContextTagKeys()`.